website/docs/api-layers.md

```diff
@@ -957,10 +933,10 @@ https://github.com/explosion/thinc/blob/master/thinc/layers/bidirectional.py
 Compose two models `f` and `g` such that they become layers of a single
 feed-forward model that computes `g(f(x))`.
 
-| Argument    | Type                             | Description                      |
-| ----------- | -------------------------------- | -------------------------------- |
-| `*layers`   | <tt>Model[ArrayXd, ArrayXd]</tt> | The models to compose.           |
-| **RETURNS** | <tt>Model[ArrayXd, ArrayXd]</tt> | The composed feed-forward model. |
+| Argument    | Type                | Description                      |
+| ----------- | ------------------- | -------------------------------- |
+| `*layers`   | <tt>Model</tt>, --- | The models to compose.           |
```

Comment:
This `---` notation is not very clear. Why don't we have a row for `layer1`, one for `layer2` and then another for the optional additional layers? Also I noticed that this PR actually renamed `*layers` to `*more_layers`. Won't that be breaking for some users?

Reply:
I'm not sure, but have changed it back to be on the safe side.